src/instructlab/model/train.py

```diff
@@ -473,22 +493,18 @@ def get_files(directory: str, pattern: str) -> list[str]:
         shutil.copy(train_files[0], train_file)
         shutil.copy(test_files[0], test_file)
 
-    if (
-        pipeline == "full"
-        or pipeline == "accelerated"
-        or strategy == SupportedTrainingStrategies.LAB_MULTIPHASE.value
-    ):
+    if pipeline == "full":
```

Comment:
this should be `accelerated or full and not multiphase`

Reply:
the library requires data-path to be a jsonl when not using phased

Reply:
just fixed this from your request